src/distributions/other.rs

```diff
@@ -36,34 +36,18 @@ use crate::Rng;
 ///
 /// # Passwords
 ///
-/// We caution that strings produced by sampling `Alphanumeric` tend not
-/// to be particularly memorable when used as passwords by humans.  
-/// Instead, we suggest that human memorable passwords be created by
-/// drawing words independently and uniformly at random from a large wordlist. 
+/// Users sometimes ask whether it is safe to use a string of random characters
+/// as a password. `Alphanumeric` generates from an alphabet of 62 symbols, thus
+/// each character can provide `log2(62) = 5.95...` bits of entropy. We suggest
+/// consulting external sources for more. One may start with the
+/// [Wikipedia article on Password Strength](https://en.wikipedia.org/wiki/Password_strength).
```

Comment:
I think it is a bit confusing to mention the question without answering it. I understand that we want to avoid making inaccurate statements about security. Maybe we should emphasize that the default RNG in Rand is designed to be unpredictable and thus suited for generating passwords, but that it is more conservative to use `getrandom` (or the equivalent randomness implementations in crypto libraries).

I would maybe emphasize that giving recommendations about the strength/length of passwords is out of scope for Rand.